wiki/How_You_Can_Help!/id.md

```diff
@@ -0,0 +1,81 @@
+# Bagaimana Anda dapat membantu!
+
+_Sebelumnya ini adalah kiriman yang [peppy](/users/2 "peppy") buat beberapa waktu lalu dan diminta untuk menjadikannya sebuah artikel di sini. Anda dapat melihat kiriman aslinya [di sini](/community/forums/topics/83155)._
+
+Saya sudah lama ingin memposting hal ini untuk sementara, tetapi selalu tertunda di saat Saya menyiapkan segala sesuatunya di situs web dan di kepala saya. Seperti yang mungkin Anda sadari, komunitas osu! beserta ekosistemnya telah tumbuh stabil, tetapi tim pengembang belum tumbuh dalam ukuran sama sekali. Saya yakin dengan komunitas sebesar ini, pasti banyak bakat tersembunyi di luar sana, jadi tujuan kiriman ini adalah untuk menjangkau dan memberi tahu Anda semua bahwa kami selalu mencari bantuan dan masukan, serta menguraikan situasi saat ini di mana Anda dapat langsung terjun dan membantu!
+
+Pertama, dari perspektif pribadi saya sudah benar-benar jenuh. Saya menghabiskan semua waktu luang saya untuk osu! (atau osu!stream) dan tidak memiliki sumber daya lain lagi yang dapat saya andalkan. Ini berarti bahwa sebagian besar hal-hal yang tersedia di osu! saat ini didasarkan pada prioritas pribadi saya. Ini bukan berarti bahwa itu harus menjadi faktor pembatas - ada begitu banyak peluang untuk membuat osu! lebih baik tanpa perlu akses atau kemampuan seorang pengembang.
+
+Berikut adalah daftar terkini hal-hal yang saya pikir semua orang dapat membantu, disortir berdasarkan dari yang paling penting di mata saya. Saya akan mencoba terus memperbarui kiriman ini ketika saya melihat hal-hal lain terpikirkan, tetapi jangan ragu untuk memberikan saya (sebaiknya di kiriman ini) jika Anda memilikinya).
+
+## Bantu pengembangan [osu!wiki](https://github.com/ppy/osu-wiki "osu-wiki di GitHub")!
+
+Untuk menjaga konten di situs web tetap segar dan terkini, saya ingin memindahkan semua bagian yang berisi teks ke wiki. Hal ini termasuk FAQ, panduan ranking (ranking guidelines), semua panduan berbasis forum kami, bagian sejarah/informasi. Meskipun kami telah membuat kemajuan besar (berkat beberapa pengguna yang sangat berdedikasi), memperbarui wiki adalah proses berkelanjutan yang pada dasarnya dapat dilakukan oleh siapa pun. Ada juga banyak kesempatan untuk melakukan terjemahan yang tersisa untuk membuat konten dapat diakses dalam bahasa lain yang dikenal umat manusia!
```

Comment:
"tetap segar dan terkini" -> "sesegar mungkin"? kayaknya dua kata "as possible" ganjal dengan "dan terkini"

Reply:
sesegar mungkin seems like a drink not an article.
let's wait another reviewer